snippets/celsius_to_fahrenheit.md

```diff
@@ -0,0 +1,19 @@
+---
+title: celsius_to_fahrenheit
+tags: conversion, beginner
+---
+
+Converts Celsius to Fahrenheit.
+
+fahrenheit = (celsius * 1.8) + 32

```

Comment:
Please make your description more like a sentence (e.g. `Use the formula ... to convert the temperature from Celsius to Fahrenheit.`)